resolwe/flow/tests/test_executors.py

```diff
@@ -151,3 +110,13 @@ def test_workflow(self):
         self.assertEqual(step2_data.input['param2']['a'], step1_data.pk)
         self.assertEqual(step2_data.input['param2']['b'], 'hello')
         self.assertEqual(step2_data.output['out1'], 'simon says: hello world')
+
+    @with_docker_executor
```

Comment:
Is there any particular reason to put these two tests in the same class as above?

I would prefer to put them in a separate class, or rename existing class to something more general.